### PR TITLE
Implementing FavoritesViewController skeleton

### PR DESCRIPTION
**Pull request by AnnaBanani**
Implementing FavoritesViewController skeleton

### CODE REVIEW

File: TheMET/TheMET/ArtView/ArtView.swift [hunk 1]
```diff
@@ -76,31 +76,35 @@ class ArtView: UIView {
...
+            self.textLabel.topAnchor.constraint(equalTo: self.likeButton.bottomAnchor, constant: 10),
+            self.textLabel.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: -20)
         ])
+//        NSLayoutConstraint.activate([
+//            self.tagView.topAnchor.constraint(equalTo: self.topAnchor),
+//            self.tagView.leadingAnchor.constraint(equalTo: self.leadingAnchor),
+//            self.tagView.trailingAnchor.constraint(equalTo: self.trailingAnchor),
+//            self.tagView.topAnchor.constraint(equalTo: self.textLabel.bottomAnchor, constant: 10),
+//            self.tagView.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: -20)
+//        ])
```
Comment (Ssimboss) [inline TheMET/TheMET/ArtView/ArtView.swift:108]: Pls create a project task to fix this tag view later, then make a comment in this code referencing this task.

File: TheMET/TheMET/ArtView/ArtViewCell.swift [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        get {return self.artView.onLikeButtonDidTap }
+        set {self.artView.onLikeButtonDidTap = newValue }
+    }
+    var tags: [String] {
+        get { return self.artView.tags }
+        set { self.artView.tags = newValue }
+    }
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+       super.init(style: style, reuseIdentifier: reuseIdentifier)
```
Comment (Ssimboss) [inline TheMET/TheMET/ArtView/ArtViewCell.swift:40]: this code is outlined

File: TheMET/TheMET/TagListView/TagListView.swift [hunk 3]
```diff
@@ -53,6 +53,8 @@ class TagListView: UIView {
     
     override init(frame: CGRect) {
         super.init(frame: frame)
+//        self.setContentHuggingPriority(UILayoutPriority(251), for: .horizontal)
```
Comment (Ssimboss) [inline TheMET/TheMET/TagListView/TagListView.swift:56]: Unnecessary changes. remove

File: TheMET/TheMET/ViewControllers/FavoritesViewController.swift [hunk 4]
```diff
@@ -20,24 +22,51 @@ class FavoritesViewController: UIViewController {
...
+        self.tableView.delegate = self
         
-       
+    }
+    
+//  UITableViewDelegate, UITableViewDataSource
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if let cell = tableView.dequeueReusableCell(withIdentifier: ArtViewCell.artViewCellIdentifier, for: indexPath) as? ArtViewCell {
+            cell.backgroundColor = .clear
```
Comment (Ssimboss) [inline TheMET/TheMET/ViewControllers/FavoritesViewController.swift:47]: Please us different random content for at least 2-3 cells to check that it layouts it OK

**Review by Ssimboss — CHANGES_REQUESTED** [review]

File: TheMET/TheMET/TagListView/TagListView.swift (hunk 3 above)
Comment (AnnaBanani) [reply TheMET/TheMET/TagListView/TagListView.swift:56]: +

File: TheMET/TheMET/ArtView/ArtView.swift (hunk 1 above)
Comment (AnnaBanani) [reply TheMET/TheMET/ArtView/ArtView.swift:108]: +

File: TheMET/TheMET/ArtView/ArtViewCell.swift (hunk 2 above)
Comment (AnnaBanani) [reply TheMET/TheMET/ArtView/ArtViewCell.swift:40]: +

File: TheMET/TheMET/ViewControllers/FavoritesViewController.swift (hunk 4 above)
Comment (AnnaBanani) [reply TheMET/TheMET/ViewControllers/FavoritesViewController.swift:47]: +

File: TheMET/TheMET/ArtView/ArtView.swift [hunk 5]
```diff
@@ -76,6 +76,7 @@ class ArtView: UIView {
         self.loadingImageView.translatesAutoresizingMaskIntoConstraints = false
         self.likeButton.translatesAutoresizingMaskIntoConstraints = false
         self.textLabel.translatesAutoresizingMaskIntoConstraints = false
+//        tagView placement problem is going to be solved in  one the next tasks.
```
Comment (Ssimboss) [inline TheMET/TheMET/ArtView/ArtView.swift:79]: Please provide the task-number.
Also you cause a `TODO:` marker to make it visible in code-hierarchy (just like `MARK:`)

**Review by Ssimboss — CHANGES_REQUESTED** [review]

File: TheMET/TheMET/ArtView/ArtView.swift (hunk 5 above)
Comment (AnnaBanani) [reply TheMET/TheMET/ArtView/ArtView.swift:79]: +

**Review by Ssimboss — APPROVED** [review]
